Adyen/Model/PaymentComponentData.swift

```diff
@@ -32,6 +32,9 @@ public struct PaymentComponentData {
     
     /// The installments object.
     public let installments: Installments?
+    
+    /// Indicates whether the current SDK version suports native redirect without glue pages.
+    public let supportNativeRedirect: Bool = true
```

Comment:
is it good idea to set it `true` by default ?
On what API version will it be available? 
Can it trigger "unknown field" on LIVE?

Reply:
if merchants want to disable it, they can just not send it in the payments call

also its not versioned, so its available for all API versions :-)

Reply:
Still don't think it is a good idea to enable experimental feature for everyone.

> if merchants want to disable it, they can just not send it in the payments call

For Sessions it is always "ON, so to opt-out merchant must be aware of this field existence.

Reply:
then merchants can disable the account data property on the backend :-), right?